Formula/pachi.rb

```diff
@@ -18,40 +18,28 @@ class Pachi < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "e0fc26989c0cf90b6fa2256e129b0b87993464ec27ad88fefe569abdd9702292"
   end
 
-  resource "patterns" do
-    url "https://sainet-dist.s3.amazonaws.com/pachi_patterns.zip"
-    sha256 "73045eed2a15c5cb54bcdb7e60b106729009fa0a809d388dfd80f26c07ca7cbc"
-  end
+  depends_on "caffe"
 
-  resource "book" do
-    url "https://gnugo.baduk.org/books/ra6.zip"
-    sha256 "1e7ffc75c424e94338308c048aacc479da6ac5cbe77c0df8adc733956872485a"
+  resource "datafiles" do
+    url "https://launchpad.net/~lemonsqueeze/+archive/ubuntu/pachi/+sourcefiles/pachi-go-data/12.70/pachi-go-data_12.70.orig.tar.xz"
+    sha256 "f907292e34d3d4e79b0ca1d6181b36fe15aad3741a02f622094657a909910a8b"
   end
 
   def install
+    ENV["DESTDIR"] = HOMEBREW_PREFIX
```

Comment:
We don't actually want to install directly into `HOMEBREW_PREFIX`, I think.

Reply:
Yeah, was not sure about this one. Maybe i should set PREFIX = HOMEBEW_PREFIX instead, this is what i intended to do.